packages/react/src/components/CardLink/README.md

```diff
@@ -47,6 +47,8 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 | `href`      | YES      | String    | n/a           | Valid URL for a the location of an internal or external resource.                                                                |
 | `icon`      | NO       | Element   | null          | Provide an optional icon to the footer from [Carbon's icon library](https://www.carbondesignsystem.com/guidelines/icons/library) |
 | `title`     | YES      | String    | n/a           | Concise yet descriptive string of text describing the linked resource.                                                           |
+| `imgSrc`    | NO       | String    | null          | Image source to be passed as a property to the to the CardLink component                                                         |
+| `altText`   | NO       | String    | null          | Image alternate text to be passed as a property to the to the CardLink component                                                 |
```

Comment:
@photodow just checking if these changes are something you are tracking for the CardLink component?

Reply:
Sure will move _matchHeight into `packages/utilities`